db/products.js

```diff
@@ -44,3 +44,9 @@ export const createProducts = async product => {
 		throw error;
 	}
 };
+
```

Comment:
My bad, forgot that the export prefix is enabled by babel which we dont use in the backend.